benchmark/storage/data_table_benchmark.cpp

```diff
@@ -71,6 +71,7 @@ BENCHMARK_DEFINE_F(DataTableBenchmark, SimpleInsert)(benchmark::State &state) {
 // Insert the num_inserts_ of tuples into a DataTable concurrently
 // NOLINTNEXTLINE
 BENCHMARK_DEFINE_F(DataTableBenchmark, ConcurrentInsert)(benchmark::State &state) {
+  MultiThreadedTestUtil mtt_util;
```

Comment:
Since it is no longer stateless, maybe rename it to `MultiThreadedTestHarness`

Reply:
Test harness has a very specific meaning to me when referring to Google Tests (something the tests inherit from). I'm not sure this applies.